modules/user_accounts/php/useraccountsprovisioner.class.inc

```diff
@@ -0,0 +1,65 @@
+<?php declare(strict_types=1);
+/**
+ * This file implements a data provisioner to get all modules
+ * for the User Accounts menu page.
+ *
+ * PHP Version 7
+ *
+ * @category   Core
+ * @package    Main
+ * @subpackage Core
+ * @author     Dave MacFarlane <dave.macfarlane@mcin.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
```

Comment:
The file comment (and most of the `@` tags in the class comment) are no longer required as of #6046